x-pack/plugins/observability/public/components/custom_threshold/components/preview_chart/preview_chart.tsx

```diff
@@ -97,13 +89,6 @@ export function PreviewChart({
   useEffect(() => {
     if (!threshold) return;
     const refLayers = [];
-    const isPercent = Boolean(metrics.length === 1 && metrics[0].field?.endsWith('.pct'));
```

Comment:
There is no need to format the threshold. The threshold should be the same as the field value, otherwise.

Reply:
I didn't understand, if we don't set the format how does Lens know how to format the value?

Reply:
As I mentioned, the threshold should follow the field value without any format. The lens will adjust the threshold line/value in the right place based on the formatting applied in the chart.

Reply:
@maciejforcone @vinaychandrasekhar I think for percent, it is better not to have any decimals in the preview chart, so this:
![image](https://github.com/elastic/kibana/assets/12370520/1d7552be-bcc3-41b5-bd97-d54c880b55f6)
Instead of this:
![image](https://github.com/elastic/kibana/assets/12370520/b98196b0-77ec-4418-9344-04a0f6ae1941)
What do you think?

Reply:
Done